### DISCUSSION TITLE
Menu drop down

### DISCUSSION THREAD

**Discussion by danchitnis** [chakra-ui/chakra-ui#4518] [Q&A]
The [Menu](https://chakra-ui.com/docs/overlay/menu) component seems to always drop down from the top left corner. Is there a way to change this, for example, drop down from top-centre? of course some menus open from bottom to top if they are located and the bottom of the screen.

**Comment by daniel-schulz-codes** [comment]
Hej @danchitnis you could change the placement of the menu with the `placement` property in the `Menu`-props. Best check the docs for available placements! :)
